index.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask, jsonify, request, abort
+from flask_cors import CORS
+
+import googleapiclient.discovery, os, config
+from comments import get_comments
+
+
+app = Flask(__name__)
+CORS(app)
+
+comment_list = []
+
+@app.route('/')
+def create_UI():
+    return "<h1 style='color:red;'>loser</h1>"
+
+@app.route('/api/ytVideoIds', methods=['POST'])
+def create_ytVideoId():
+
+    if not request.json or not 'videoId' in request.json:
+        abort(400)
+
+    yt_video_id = request.get_json('videoId')
+    comment_data = get_comments(yt_video_id) # from line 12, it gets "Bh_uMYaykyQ"
+
+    return jsonify({'ytVideoId': yt_video_id}), 201
+
+def sanitize(data):
+    sanitized_data = YTComment(data)
+    return sanitized_data
+
+for comment in comment_data["items"]:
+    comment_list.append(sanitize(comment))
+
+comment_list = sort(comment_list)
```

Comment:
Why not just sort it in the for-loop above?

Reply:
Good point

Reply:
Since the for-loop is creating comment_list (as it is empty before the for-loop starts), I am not sure how I would sort during while the comment_list values are being created without another for-loop. For example, if the loop just finished its second iteration and the sentiment values were [0, 2], and the third iteration wants to add a sentiment value of 1, it would have to loop through the list to add it in the correct place. And I can't just plop it in the middle statically since the list may eventually be 25 values, and plopping it in the middle would not necessarily be the right place.

Therefore, sentiment_sort() iterates after the list is done being sorted